engines/gob/inter_v5.cpp

```diff
@@ -103,8 +103,7 @@ void Inter_v5::o5_deleteFile() {
 	if (mode == SaveLoad::kSaveModeSave) {
 
 		if (!_vm->_saveLoad->deleteFile(file)) {
-			GUI::MessageDialog dialog(_("Failed to delete file."));
-			dialog.runModal();
+			warning("Cannot delete file \"%s\"", file);
```

Comment:
Why do we now print something into the console instead of giving feedback to the user?

Reply:
In Adibou 2, the scripts sometimes call deleteFile() without checking whether the file does exist, so this feedback was too noisy. I can revert it here however, since this "o5_deleteFile" has been finally reimplemented in Adibou classes (o7_deleteFile).